### PR TITLE
Tweak some german language strings

### PR DESCRIPTION
**Pull request by blubbfish**
Change some string to get a better wording in german

### CODE REVIEW

**Comment by blubbfish** [comment]
@MarlinFirmware/language-team so now we can tweak some strings :)

File: Marlin/language_de.h [hunk 1]
```diff
@@ -131,7 +131,7 @@
 #define MSG_RESUMING                        "Druckfortsetzung"
 #define MSG_PRINT_ABORTED                   "Druck abgebrochen"
 #define MSG_NO_MOVE                         "Motoren eingeschaltet"
-#define MSG_KILLED                          "KILLED"
+#define MSG_KILLED                          "ABGEBROCHEN"
 #define MSG_STOPPED                         "ANGEHALTEN"
 #define MSG_CONTROL_RETRACT                 "Retract mm"
```
Comment (blubbfish) [inline Marlin/language_de.h:136]: The word Retract is not german. Maybe we can use something with "zurückziehen" or "einziehen"

File: Marlin/language_de.h [hunk 2]
```diff
@@ -147,7 +147,7 @@
 #define MSG_ZPROBE_OUT                      "Sensor ausserhalb"
 #define MSG_HOME                            "Vorher"  // Used as MSG_HOME " " MSG_X MSG_Y MSG_Z " " MSG_FIRST
 #define MSG_FIRST                           "homen"
```
Comment (blubbfish) [inline Marlin/language_de.h:149]: That is also not nice. I would prefere something with "Startposition"

File: Marlin/language_de.h (hunk 1 above)
Comment (thinkyhead) [reply Marlin/language_de.h:136]: Tricky when it comes to this specific domain, 3D printing.
Umkehren? Rückwärts?

**Pull request merged**